christfellowshipchurch/web-app-v2
# Storybook Relaunch

### About
This PR updates and adds our components and ui-kit elements to storybook for better and more consistent testing.

### Test Instructions
1. Run `yarn && yarn storybook` on the terminal (on the `storybook-relaunch` branch), it should open the storybook page once it finishes loading, but if it does not, go to [localhost:6006](http://localhost:6006/?path=/docs/configure-your-project--docs).
2. On the left you will see our components and ui-kit elements, ensure all of the components and ui-kit elements that are there work as expected.

### Screenshots
<img width="1440" alt="Screenshot 2023-10-23 at 11 13 19 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/ddad9019-a074-45ae-a957-e73dad86bf23">

### Closes Tickets
[CFDP-2789](https://christfellowshipchurch.atlassian.net/browse/CFDP-2789)


[CFDP-2789]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2789?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### More To Add
Missing UI: 	
	* Horizontal Scroll - still has not been merged into main 
        * HTML Render & Img
	* Modal
	* Theme Provider
	* Value Stack

Missing Components:
	* Community Single
	* ContentBlockFeature
	* Content Layout
	* Content List
	* Content Single
	* Custom Link
	* Discover Filter Section
	* Discover Filter Map
	* Event Single
	* External Home
	* Feature Feed
	* Gender Field
	* Generic Error
	* Gradient Background
	* Group Email Component
	* Group Manage
	* Groups Single
	* Groups List
	* Group Search Filters
	* Groups Search Side Bar
	* Group Resource List
	* Hero Landing
	* Home Feed
	* Info Card List
	* Layout
	* Legacy Node Router
	* Live Steam Single
	* Location Block Feature
	* Location Single
	* Logo
	* Mobile Nav Screen
	* MODALS ******** A lot of these inside
	* Nav
	* Not Found
	* Page Single
	* Password Reset
	* Search Field
	* Share
	* Testimonials
	* Track Event When Loaded
	* User Avatar
	* User Profile
	* Video
	* Wistia Player
